content/course-section/assignments.md

```diff
@@ -0,0 +1,9 @@
+menu: 
```

Comment:
Does using yaml prevent us from having newlines in the HTML?

Reply:
Just made a commit that cleans up the sample HTML